dex/testing/dcr/harness.sh

```diff
@@ -302,50 +302,54 @@ ENABLE_VOTING="2" # 2 = enable voting and ticket buyer
 MANUAL_TICKETS="0"
 "${HARNESS_DIR}/create-wallet.sh" "$SESSION:3" "alpha" ${ALPHA_WALLET_SEED} \
 ${ALPHA_WALLET_RPC_PORT} ${USE_SPV} ${ENABLE_VOTING} ${ALPHA_WALLET_HTTPPROF_PORT} \
-${MANUAL_TICKETS}
+${MANUAL_TICKETS} "_"
 # alpha uses walletpassphrase/walletlock.
 
 # SPV wallets will declare peers stalled and disconnect with only ancient blocks
 # from the archive, so we must mine a couple blocks first, but only now after the
 # voting wallet (alpha) is running.
 tmux send-keys -t $SESSION:0 "./mine-alpha 2${WAIT}" C-m\; wait-for donedcr
 
+echo "Creating simnet vspd wallet"
+USE_SPV="0"
+ENABLE_VOTING="1"
+MANUAL_TICKETS="1"
+"${HARNESS_DIR}/create-wallet.sh" "$SESSION:7" "vspdwallet" ${VSPD_WALLET_SEED} \
+${VSPD_WALLET_RPC_PORT} ${USE_SPV} ${ENABLE_VOTING} "_" ${MANUAL_TICKETS} "_"
+
+echo "Creating simnet vspd"
+ALPHA_WALLET_PUBKEY=$("${NODES_ROOT}/harness-ctl/alpha" "getmasterpubkey")
+"${HARNESS_DIR}/create-vspd.sh" "$SESSION:8" "${VSPD_PORT}" "${ALPHA_WALLET_PUBKEY}"
+
```

Comment:
I had to update the vspd commit in **create-vspd.sh**, otherwise vspd crashed.

Reply:
hmm, its crashing for me after updating

Reply:
Oh, I had to update my system's vspd binary

Reply:
Updated to latest release version `release-v1.3.2`